packages/iris-grid/src/IrisGridUtils.test.ts

```diff
@@ -661,6 +661,8 @@ describe('dehydration methods', () => {
     expect(
       // This makes sure the result doesn't contain undefined
       // so it can be serialized and de-serialized without changes
+      // Note we're using the `deep-equal` package here for strict deep comparison, so we know if there's an undefined.
```

Comment:
Does fast-deep-equal not do this?

Reply:
No.

Reply:
I guess I'm ok with this as worst case we're making implicit/explicit undefined interchangeable and might have saves where something changes from implicit to explicit in the code being switched to `fast-deep-equal`

Reply:
To be clear, the default behaviour between the two is the same. Deep-equal is continued to be used here in the tests because it adds the strict: true flag, which is not available in fast-deep-equal.